### PR TITLE
Personal Plan: Remove hardcoded data

### PR DESCRIPTION
**Pull request by rodrigoi**
This PR removes the hardcoded data for Personal Plan and changes the endpoint version so it uses only backend data.

~~This depends on Diff D2267.~~

Test live: https://calypso.live/?branch=update/personal-plan-cleanup

### CODE REVIEW

**Comment by rodrigoi** [comment]
Ready for review. Diff 2264 has been deployed fixing the discount bug and returning the Personal Plan only version `v1.4` of '/plans`.

This will fix the _Free Domain Available_ bug and allow the Personal Plan to be enabled on all environments (including Desktop).

cc: @roundhill @apeatling @rralian @lamosty

**Comment by roundhill** [comment]
Looks good, I saw the personal plan being returned in the `v1.4` api response and the plans pages worked correctly.

:shipit:

File: client/state/plans/test/fixture.js [hunk 1]
```diff
@@ -254,7 +252,7 @@ export const PLAN_2002 = {
 };
 
 export const WPCOM_RESPONSE = [
-	PLAN_1, PLAN_1003, PLAN_1008, PLAN_1009, PLAN_2000, PLAN_2001, PLAN_2002
+	PLAN_1, PLAN_1003, PLAN_1008, PLAN_2000, PLAN_2001, PLAN_2002
```
Comment (lamosty) [inline client/state/plans/test/fixture.js]: Why we don't have `PLAN_1009` (= Personal plan) in `WPCOM_RESPONSE` array? Shouldn't it list all the plans included in the API response?

**Comment by lamosty** [comment]
Except for the one question, this looks good to me. :+1:

File: client/state/plans/test/fixture.js (hunk 1 above)
Comment (rodrigoi) [inline client/state/plans/test/fixture.js]: it should. I've added the personal plan to the test fixture so `nock` returns all the plans.

**Pull request merged**